### ISSUE TITLE
SIM-OPS Add kubernetes support

### ISSUE THREAD

**Issue by YeagerAI-Bulat** [yeagerai/genlayer-simulator#244]
Need to add Kubernetes deployment support using Helm chart to host it in the cloud

**Comment by dbfreem** [comment]
So is the idea here is that you would want a helm chart that basically produces what you have in the docker compose but for k8s?

**Comment by YeagerAI-Bulat** [comment]
Yes, exactly! We will create k8s helm chart, but not now, since this it's not our top priority.
